source/GameData.cpp

```diff
@@ -283,6 +283,14 @@ const vector<string> &GameData::Sources()
 
 
 
+// Get a reference to the UniverseObjects object.
+UniverseObjects &GameData::Objects()
```

Comment:
I was actually thinging of (re)using the getter for Missions, but this also works.

Reply:
`GameData::Missions()` returns `const`, so I can't add a new mission.